src/components/GetInvolved.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import '../App.css';
+
+/*
+class GetInvolved extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+  componentDidMount() {}
+  componentDidUpdate() {}
+
+  render() {
+    return (
+      <div className="dataview-container">
+        <h2>Get Involved</h2>
+        <p>This page is currently under development.</p>
+      </div>
+    );
+  }
+}
+
+<li><b> :</b> Poll for student opinion in an effort to gather feedback and observe usage patterns. Take Student Name, ID and an "I approve" box?Captcha?Enforce 1 submission/IP?</li>
+*/
+
+const GetInvolved = props => (
+  <div className="container">
+    <div className="dataview-container">
+      <h1 style={{marginBottom: ".75em"}}><b>Get Involved</b></h1>
+      <p>Thanks for your interest in contributing to STEV! As this project continues to grow, there are increasingly many opportunities for involvement. Those with engineering backgrounds are encouraged to jump into the code and submit a pull request (more on that <a href="#pull-requests">below</a>). No matter your background, there are a number of ways to contribute to the project. A few recent challenges are listed here. </p>
+      <h3>Current Contribution Suggestions</h3>
+      <p><i>Updated Spring 2019</i></p>
+      <ul>
+        <li><b>Bug Tracking:</b> Submit a bug you discovered! Or submit an issue regarding the lack of data. If you discover a bug, make sure to jot down the steps to reproduce and submit a bug report in the "Issues" tab on our <a href="https://github.com/stev-ou">GitHub</a>. The latter contribution is especially valuable since our team has attempted to work with the university many times over, with very little receptiveness or action on the university's behalf. As we collect more information about missing data, and the demand for rectifying it, we will continue our attempts to gather more data from the university.</li>
```

Comment:
"As we collect more information about the demand for missing evaluations, we will continue ..." in last sentence